docs/Install.md

```diff
@@ -1,22 +1,125 @@
 
 # Installation
 
-The following will walk through the deployment of Pelorus.
+There are three methods to deploy Pelorus:
+
+(Recommended) First two are using Pelorus Operator that is available by default in the [Red Hat-provided community-operators catalog sources](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs). This catalog is distributed as part of the OpenShift:
+
+  * OpenShift [Command Line Tool](#openshift-command-line-tool)
+  * OpenShift [Dashboard](#openshift-dashboard)
+
+Third one is using Helm3 charts method. It is recommended for development of Pelorus only:
+
+  * [Helm charts](#helm-charts)
 
 ## Prerequisites
 
 Before deploying the tooling, you must have the following prepared
 
-* An OpenShift 4.7 or higher Environment
-* A machine from which to run the install (usually your laptop)
-  * The OpenShift Command Line Tool (oc)
+* Access to an OpenShift 4.7 or higher cluster via [Dashboard](#openshift-dashboard) or [CLI](#openshift-command-line-tool).
+
+* ***CLI*** access requires a machine from which to run the install with an OpenShift Command Line Tool (oc)
+
+* ***Helm charts*** requires additional tools on your system:
+    * [Helm3](https://github.com/helm/helm/releases)
+    * jq
+    * git
+
+## OpenShift Command Line Tool
+
+### Installing Operator
+
+Installing Pelorus using CLI is equivalent to the [OpenShift Dashboard](#openshift-dashboard) method.
+
+1. Create pelorus namespace. You can use other namespace to which Pelorus will be installed, however we do recommend using default one, which is **pelorus**.
```

Comment:
I am not a fan of adding bullets to the text, I would simply let the text without them. See here to see with you prefer without bullets https://pelorus--759.org.readthedocs.build/en/759/getting_started/Install/#initial-deployment